.github/workflows/check-jax-release.yaml

```diff
@@ -84,6 +84,9 @@ jobs:
 
     - name: Build Frontend
       run: |
+        # TODO: remove workaround before merging to main
+        pip install pennylane-lightning==0.38
+        pip install git+https://github.com/PennyLaneAI/pennylane@v0.38.0-rc0
```

Comment:
I think this action is more or less abandoned and probably doesn't need this patch
```suggestion
```

Reply:
Done